compiler/symbol.go

```diff
@@ -342,22 +373,17 @@ func (c *compilerContext) parsePragmas(info *functionInfo, f *ssa.Function) {
 	}
 }
 
-// Check whether this function cannot be used in //go:wasmimport. It will add an
-// error if this is the case.
+// Check whether this function can be used in //go:wasmimport or
+// //go:wasmexport. It will add an error if this is not the case.
 //
 // The list of allowed types is based on this proposal:
 // https://github.com/golang/go/issues/59149
-func (c *compilerContext) checkWasmImport(f *ssa.Function, pragma string) {
+func (c *compilerContext) checkWasmImportExport(f *ssa.Function, pragma string) {
 	if c.pkg.Path() == "runtime" || c.pkg.Path() == "syscall/js" || c.pkg.Path() == "syscall" {
 		// The runtime is a special case. Allow all kinds of parameters
 		// (importantly, including pointers).
 		return
 	}
-	if f.Blocks != nil {
```

Comment:
This should probably be retained for `go:wasmimport`

Reply:
It is, the check has been moved into `case "//go:wasmimport":` above.